lib/finch.ex

```diff
@@ -57,6 +57,25 @@ defmodule Finch do
       used to configure proxying, https settings, or connect timeouts.
       """,
       default: []
+    ],
+    pool_max_idle_time: [
+      type: :timeout,
+      doc: """
+      The maximum number of milliseconds that a pool can be idle before being terminated, used only by HTTP1 pools. \
+      This options is forwarded to NimblePool and it starts and idle verification cycle that may impact \
+      performance if misused. For instance setting a very low timeout may lead to pool restarts. \
+      Defaults to no timeout. \
```

Comment:
NimbleOptions will automatically add a line that states the default, you can see here: https://hexdocs.pm/finch/Finch.html#start_link/1-pool-configuration-options
```suggestion
```